packages/components/src/components/carousel/carousel.stories.ts

```diff
@@ -2,14 +2,21 @@
 import '../../solid-components';
 import { html } from 'lit';
 import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
-import { waitUntil } from '@open-wc/testing-helpers';
 
 const { argTypes, parameters } = storybookDefaults('sd-carousel');
 const { overrideArgs } = storybookHelpers('sd-carousel');
 const { generateTemplate } = storybookTemplate('sd-carousel');
 
+/**
+ * Carousels display an arbitrary number of content slides along a horizontal axis.
```

Comment:
Displays an arbitrary number of content slides along a horizontal axis.

Reply:
Updated

Reply:
Updated